db/re/item_db.txt

```diff
@@ -12624,6 +12638,9 @@
 31476,C_Cherry_Blossom_Hat_YL,C Yellow Cherry Blossom Hat,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1727,{},{},{}
 31478,C_Eleanor_Wig_,C Eleanors Wig Lower,4,0,,0,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1502,{},{},{}
 //
+31479,C_Group_of_Stars,Costume Group of Stars,4,,,0,,,,,0xFFFFFFFF,63,2,4096,,1,0,1729,{ hateffect(HAT_EF_MANYSTARS,true); },{},{ hateffect(HAT_EF_MANYSTARS,false); }
+31588,C_Snow_Flower,C Snow Flower,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1799,{ hateffect(HAT_EF_SNOWFLAKE_TIARA,true); },{},{ hateffect(HAT_EF_SNOWFLAKE_TIARA,false); }
+31602,C_Midgarts_Glory,C Rune Midgarts Glory,,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,,1807,{ hateffect(HAT_EF_MIDGARTS_GLORY,true); },{},{ hateffect(HAT_EF_MIDGARTS_GLORY,false); }
```

Comment:
Type should be 12(Shadow Equipment) not 0(Healing item)

Reply:
This was fixed [here](884043b).